src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointEvidence.java

```diff
@@ -184,15 +254,7 @@ protected void serialize( final Kryo kryo, final Output output ) {
 
         @Override
         public String toString() {
-            final StringBuffer out = new StringBuffer();
-            out.append(super.toString());
-            out.append("\t");
-            out.append("Split");
-            out.append("\t");
-            out.append(cigar);
-            out.append("\t");
-            out.append((tagSA == null ? " SA: None" : (" SA: " + tagSA)));
-            return out.toString();
+            return super.toString()+"\tSplit\t"+cigar+"\t"+(tagSA == null ? " SA: None" : (" SA: " + tagSA));
```

Comment:
Mindless consistency.  All the other classes implement it this way.
As far as efficiency goes, they are completely equivalent, I believe.  So it's a matter of clarity.  To my eye the concatenation is a little clearer, but YMMV.  Happy to change it back if you think the StringBuffer way is clearer.